bazel/external/wee8.patch

```diff
@@ -53,3 +54,14 @@
 
  if (use_libfuzzer && is_linux) {
    if (is_asan) {
+--- wee8/third_party/depot_tools/ninja
++++ wee8/third_party/depot_tools/ninja
```

Comment:
`third_party/depot_tools/ninja` isn't called when `uname -m == aarch64`, so this shouldn't be needed.

Reply:
Yes, correct. Thanks for your comments. It has been modified.